tests/platform_tests/api/test_psu.py

```diff
@@ -319,10 +323,17 @@ def test_master_led(self, duthosts, enum_rand_one_per_hwsku_hostname, localhost,
             1: "normal",
             2: "off"
         }
-
+        supported_colors = []
         if self.num_psus == 0:
             pytest.skip("No psus found on device skipping for device {}".format(duthost))
         else:
+            if duthost.facts.get("chassis").get("master_psu_led_color"):
```

Comment:
similar to above comment, can optimize instead of calling twice duthost.facts.get("chassis").get("master_psu_led_color")

Reply:
Addressed in the last commit